prowler/providers/aws/services/opensearch/opensearch_service.py

```diff
@@ -156,7 +160,7 @@ class OpenSearchDomain(BaseModel):
     region: str
     arn: str = None
     logging: list[PublishingLoggingOption] = []
-    endpoint_vpc: str = None
+    vpc_endpoints: list[str] = None
```

Comment:
This name could be confusing, it is referring to VPC Endpoints?

Reply:
Yes, it is referring VPC Endpoints used by OpenSearch domain.